[2022-10-24T00:06:38.370+0000] {processor.py:156} INFO - Started process (PID=26) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:06:38.372+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:06:38.374+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:06:38.374+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:06:38.419+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:06:38.413+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:06:38.430+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:06:38.548+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.181 seconds
[2022-10-24T00:07:08.760+0000] {processor.py:156} INFO - Started process (PID=27) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:08.761+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:07:08.763+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:07:08.762+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:08.811+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:07:08.803+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:07:08.814+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:08.938+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.181 seconds
[2022-10-24T00:07:39.092+0000] {processor.py:156} INFO - Started process (PID=28) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:39.093+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:07:39.094+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:07:39.094+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:39.129+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:07:39.121+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:07:39.131+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:07:39.223+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.133 seconds
[2022-10-24T00:08:09.498+0000] {processor.py:156} INFO - Started process (PID=29) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:09.499+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:08:09.499+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:08:09.499+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:09.536+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:08:09.528+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:08:09.539+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:09.630+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.135 seconds
[2022-10-24T00:08:40.044+0000] {processor.py:156} INFO - Started process (PID=30) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:40.045+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:08:40.045+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:08:40.045+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:40.092+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:08:40.083+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:08:40.096+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:08:40.186+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.145 seconds
[2022-10-24T00:09:10.459+0000] {processor.py:156} INFO - Started process (PID=31) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:10.460+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:09:10.460+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:09:10.460+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:10.494+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:09:10.486+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:09:10.496+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:10.586+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.130 seconds
[2022-10-24T00:09:40.996+0000] {processor.py:156} INFO - Started process (PID=32) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:40.997+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:09:40.998+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:09:40.998+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:41.031+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:09:41.023+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:09:41.034+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:09:41.122+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.128 seconds
[2022-10-24T00:10:11.392+0000] {processor.py:156} INFO - Started process (PID=33) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:11.393+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:10:11.394+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:10:11.394+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:11.435+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:10:11.426+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:10:11.439+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:11.523+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.133 seconds
[2022-10-24T00:10:41.877+0000] {processor.py:156} INFO - Started process (PID=34) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:41.878+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:10:41.879+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:10:41.879+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:41.922+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:10:41.912+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:10:41.926+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:10:42.024+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.151 seconds
[2022-10-24T00:11:12.123+0000] {processor.py:156} INFO - Started process (PID=35) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:12.124+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:11:12.124+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:11:12.124+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:12.162+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:11:12.152+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:11:12.165+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:12.254+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.134 seconds
[2022-10-24T00:11:42.836+0000] {processor.py:156} INFO - Started process (PID=36) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:42.837+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:11:42.838+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:11:42.838+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:42.879+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:11:42.870+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:11:42.883+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:11:42.979+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.145 seconds
[2022-10-24T00:12:13.095+0000] {processor.py:156} INFO - Started process (PID=37) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:13.095+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:12:13.096+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:12:13.096+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:13.136+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:12:13.126+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:12:13.140+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:13.238+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.147 seconds
[2022-10-24T00:12:43.474+0000] {processor.py:156} INFO - Started process (PID=38) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:43.474+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:12:43.476+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:12:43.476+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:43.511+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:12:43.503+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:12:43.515+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:12:43.604+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.134 seconds
[2022-10-24T00:13:13.723+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:13.724+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:13:13.725+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:13:13.725+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:13.762+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:13:13.752+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:13:13.766+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:13.875+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.156 seconds
[2022-10-24T00:13:44.097+0000] {processor.py:156} INFO - Started process (PID=40) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:44.098+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:13:44.099+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:13:44.099+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:44.143+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:13:44.132+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:13:44.147+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:13:44.234+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-24T00:14:14.355+0000] {processor.py:156} INFO - Started process (PID=41) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:14.356+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:14:14.357+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:14:14.357+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:14.394+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:14:14.384+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:14:14.398+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:14.492+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-24T00:14:44.769+0000] {processor.py:156} INFO - Started process (PID=42) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:44.770+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:14:44.770+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:14:44.770+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:44.811+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:14:44.802+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:14:44.815+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:14:44.909+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.142 seconds
[2022-10-24T00:15:15.292+0000] {processor.py:156} INFO - Started process (PID=43) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:15.292+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:15:15.293+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:15:15.293+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:15.334+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:15:15.324+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:15:15.338+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:15.429+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-24T00:15:45.878+0000] {processor.py:156} INFO - Started process (PID=44) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:45.879+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:15:45.879+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:15:45.879+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:45.919+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:15:45.909+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:15:45.922+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:15:46.011+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.136 seconds
[2022-10-24T00:16:16.136+0000] {processor.py:156} INFO - Started process (PID=45) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:16.137+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:16:16.139+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:16:16.139+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:16.176+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:16:16.165+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:16:16.179+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:16.271+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.138 seconds
[2022-10-24T00:16:46.833+0000] {processor.py:156} INFO - Started process (PID=46) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:46.834+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:16:46.835+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:16:46.835+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:46.874+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:16:46.864+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:16:46.877+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:16:46.974+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.143 seconds
[2022-10-24T00:17:17.090+0000] {processor.py:156} INFO - Started process (PID=47) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:17.090+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:17:17.091+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:17:17.091+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:17.127+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:17:17.117+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:17:17.131+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:17.224+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.138 seconds
[2022-10-24T00:17:47.464+0000] {processor.py:156} INFO - Started process (PID=48) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:47.465+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:17:47.466+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:17:47.466+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:47.503+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:17:47.495+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:17:47.506+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:17:47.597+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.135 seconds
[2022-10-24T00:18:17.720+0000] {processor.py:156} INFO - Started process (PID=49) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:17.721+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:18:17.723+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:18:17.722+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:17.762+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:18:17.752+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:18:17.766+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:17.867+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.149 seconds
[2022-10-24T00:18:48.081+0000] {processor.py:156} INFO - Started process (PID=50) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:48.082+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:18:48.083+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:18:48.083+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:48.118+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:18:48.110+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:18:48.121+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:18:48.203+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.125 seconds
[2022-10-24T00:19:18.351+0000] {processor.py:156} INFO - Started process (PID=51) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:18.352+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:19:18.353+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:19:18.353+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:18.391+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:19:18.381+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:19:18.395+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:18.492+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.144 seconds
[2022-10-24T00:19:48.692+0000] {processor.py:156} INFO - Started process (PID=52) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:48.693+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:19:48.694+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:19:48.694+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:48.732+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:19:48.721+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:19:48.736+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:19:48.836+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.146 seconds
[2022-10-24T00:20:18.980+0000] {processor.py:156} INFO - Started process (PID=53) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:18.981+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:20:18.983+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:20:18.982+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:19.020+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:20:19.011+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:20:19.024+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:19.120+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.142 seconds
[2022-10-24T00:20:49.326+0000] {processor.py:156} INFO - Started process (PID=54) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:49.326+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:20:49.327+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:20:49.327+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:49.365+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:20:49.355+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:20:49.368+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:20:49.457+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.134 seconds
[2022-10-24T00:21:19.609+0000] {processor.py:156} INFO - Started process (PID=55) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:19.610+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:21:19.611+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:21:19.611+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:19.650+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:21:19.641+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:21:19.654+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:19.747+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.141 seconds
[2022-10-24T00:21:50.263+0000] {processor.py:156} INFO - Started process (PID=56) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:50.264+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:21:50.265+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:21:50.265+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:50.303+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:21:50.293+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:21:50.306+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:21:50.399+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.138 seconds
[2022-10-24T00:22:20.513+0000] {processor.py:156} INFO - Started process (PID=57) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:20.515+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:22:20.516+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:22:20.516+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:20.552+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:22:20.540+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:22:20.556+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:20.650+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-24T00:22:50.892+0000] {processor.py:156} INFO - Started process (PID=58) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:50.893+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:22:50.894+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:22:50.894+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:50.936+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:22:50.926+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:22:50.941+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:22:51.038+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.148 seconds
[2022-10-24T00:23:21.153+0000] {processor.py:156} INFO - Started process (PID=59) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:21.154+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:23:21.154+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:23:21.154+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:21.192+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:23:21.182+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:23:21.195+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:21.292+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.142 seconds
[2022-10-24T00:23:51.535+0000] {processor.py:156} INFO - Started process (PID=60) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:51.536+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:23:51.537+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:23:51.537+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:51.574+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:23:51.565+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:23:51.578+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:23:51.669+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.136 seconds
[2022-10-24T00:24:21.797+0000] {processor.py:156} INFO - Started process (PID=61) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:21.798+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:24:21.799+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:24:21.799+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:21.833+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:24:21.825+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:24:21.835+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:21.937+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.144 seconds
[2022-10-24T00:24:52.167+0000] {processor.py:156} INFO - Started process (PID=62) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:52.168+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:24:52.169+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:24:52.169+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:52.215+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:24:52.206+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:24:52.218+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:24:52.306+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.141 seconds
[2022-10-24T00:53:12.144+0000] {processor.py:156} INFO - Started process (PID=65) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:53:12.155+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:53:12.160+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:53:12.159+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:53:12.258+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:53:12.244+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:53:12.263+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:53:12.482+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.349 seconds
[2022-10-24T00:47:10.493+0000] {processor.py:156} INFO - Started process (PID=27) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:10.494+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:47:10.494+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:47:10.494+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:10.533+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:47:10.524+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:47:10.536+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:10.637+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.147 seconds
[2022-10-24T00:47:40.793+0000] {processor.py:156} INFO - Started process (PID=28) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:40.795+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:47:40.796+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:47:40.796+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:40.846+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:47:40.835+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:47:40.849+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:47:40.945+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.156 seconds
[2022-10-24T00:48:11.257+0000] {processor.py:156} INFO - Started process (PID=29) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:11.258+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:48:11.259+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:48:11.258+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:11.298+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:48:11.288+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:48:11.302+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:11.395+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-24T00:48:41.607+0000] {processor.py:156} INFO - Started process (PID=30) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:41.608+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:48:41.609+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:48:41.609+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:41.646+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:48:41.637+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:48:41.648+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:48:41.741+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.137 seconds
[2022-10-24T00:49:12.062+0000] {processor.py:156} INFO - Started process (PID=31) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:49:12.063+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-24T00:49:12.064+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:49:12.064+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:49:12.094+0000] {logging_mixin.py:117} INFO - [2022-10-24T00:49:12.087+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-24T00:49:12.097+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-24T00:49:12.195+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.137 seconds
